### PR TITLE
Added Amerigo game

### PR DESCRIPTION
**Pull request by chinmay7016**
#269

Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by chinmay7016** [comment]
@edm00se if you like this changes pls merge this pr

File: readme.md [hunk 1]
```diff
@@ -2517,6 +2517,20 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 5 |       13 |  90m   |
 
+### [Amerigo](https://boardgamegeek.com/boardgame/137408/amerigo)
+
+> In Amerigo, the players help Amerigo Vespucci on his journey to discover new land. The players explore the islands of South America, secure trading routes, and build settlements.
+
+The actions available to players are determined through the use of a specialized cube tower, which has appeared in the Queen titles Im Zeichen des Kreuzes and Wallenstein. At the start of the game, this tower is seeded with action cubes, which come in seven colors, with each color matching a particular type of action. During the game players will drop additional action cubes into the tower – but some of these cubes might get stuck in the floors of the tower while other cubes already in the tower are knocked free. Thus, players need to play both tactically – taking advantage of the actions currently available in the best way possible – and strategically – using their knowledge of which actions do what to play well over the course of the game.
```
Comment (edm00se) [inline readme.md:2524]: We’ll need another update to the description block here.

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update the description block formatting.

**Comment by chinmay7016** [comment]
@edm00se  sir pls review changes and merge it pls sir.

**Comment by chinmay7016** [comment]
@edm00se sir pls review changes and merge it pls sir.

**Review by edm00se — APPROVED** [review]

**Pull request merged**